scene/gui/control.cpp

```diff
@@ -462,6 +462,16 @@ bool Control::is_layout_rtl() const {
 	}
 }
 
+void Control::set_auto_translate(bool p_enabled) {
+	data.auto_translate = p_enabled;
```

Comment:
Maybe add `if (data.auto_translate != p_enabled) { ... }` to avoid notifications if nothing is changed.